next.config.js

```diff
@@ -18,4 +18,27 @@ module.exports = {
     NEXT_CONTENT_API: contentURL,
   },
   reactStrictMode: true,
+  //
+  // i18n setup
+  //
+  i18n: {
+    locales: ['en', 'fr'],
+    defaultLocale: 'en',
+    localDetection: true,
+  },
+  //
+  // rewrites setup
+  //
+  async rewrites() {
+    return [
+      {
+        source: '/accueil',
```

Comment:
This looks to work well. Seems like /fr/home still works. Any chance we could do a demo?

Reply:
yes, this is part of DC-34